x-pack/plugins/security/public/authentication/login/components/login_form/login_form.tsx

```diff
@@ -229,7 +229,9 @@ export class LoginForm extends Component<LoginFormProps, State> {
               name="username"
               data-test-subj="loginUsername"
               value={this.state.username}
-              onChange={this.onUsernameChange}
+              onChange={
+                !this.isLoadingState(LoadingStateType.None) ? undefined : this.onUsernameChange
+              }
```

Comment:
Should this also be applied to the password field below?

Reply:
I added this after searching kibana for inputs named `username` with a conditional `disabled`. But looking closer at this case, I don't think it's needed here or the password input below since they are only ever disabled after the form is submitted. Also this should be fixed for every input in eui at some point. See https://github.com/elastic/eui/issues/8238

Reply:
```suggestion
              onChange={this.onUsernameChange}
```